<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemsGroupBox</class>
 <widget class="QGroupBox" name="ItemsGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QPushButton" name="prevBtn">
     <property name="text">
      <string>..</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ballBtn">
     <property name="text">
      <string>Ball</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bumperBtn">
     <property name="text">
      <string>Bumper</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="coinBtn">
     <property name="text">
      <string>Coin</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="flipperBtn">
     <property name="text">
      <string>Flipper</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="plungerBtn">
     <property name="text">
      <string>Plunger</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="slingshotBtn">
     <property name="text">
      <string>Slingshot</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
